### ISSUE TITLE
Getting "'bind9' was not found" issue while performing Start Server and Ingest Sample Data in Cypress workflow

### ISSUE THREAD

**Issue by kshinde2512** [open-metadata/OpenMetadata#8610]
**Affected module**
Ingestion

**Describe the bug**
Getting "'bind9' was not found" issue while performing Start Server and Ingest Sample Data in Cypress workflow

**To Reproduce**
```./docker/run_local_docker.sh -d postgresql```
```./docker/run_local_docker.sh -d mysql```

**Expected behavior**
Data should be ingested.

**Additional context**
```#31 3.882 E: Version '1:9.18.4-2~bpo11+1' for 'bind9' was not found
failed to solve: executor failed running [/bin/bash -o pipefail -o errexit -o nounset -o nolog -c apt-get update     && apt-get install -t bullseye-backports -y     curl     libpcre2-8-0     postgresql-common     expat     bind9=1:9.18.4-2~bpo11+1]: exit code: 100
#31 ERROR: executor failed running [/bin/bash -o pipefail -o errexit -o nounset -o nolog -c apt-get update     && apt-get install -t bullseye-backports -y     curl     libpcre2-8-0     postgresql-common     expat     bind9=1:9.18.4-2~bpo11+1]: exit code: 100
------
 > [local-metadata-ingestion  6/20] RUN apt-get update     && apt-get install -t bullseye-backports -y     curl     libpcre2-8-0     postgresql-common     expat     bind9=1:9.18.4-2~bpo11+1:
#31 0.605 Get:9 https://packages.microsoft.com/debian/11/prod bullseye/main amd64 Packages [96.5 kB]
#31 0.749 Get:10 http://deb.debian.org/debian bullseye/main amd64 Packages [8184 kB]
#31 0.887 Get:11 http://deb.debian.org/debian bullseye-updates/main amd64 Packages [14.6 kB]
#31 1.017 Get:12 http://deb.debian.org/debian bullseye-backports/main amd64 Packages [361 kB]
#31 1.147 Get:13 http://security.debian.org/debian-security bullseye-security/main amd64 Packages [194 kB]
#31 1.276 Get:14 http://repo.mysql.com/apt/debian bullseye/mysql-8.0 amd64 Packages [8522 B]
#31 1.298 Get:15 https://apt.postgresql.org/pub/repos/apt bullseye-pgdg InRelease [91.7 kB]
#31 1.808 Get:16 https://apt.postgresql.org/pub/repos/apt bullseye-pgdg/main amd64 Packages [262 kB]
#31 3.882 E: Version '1:9.18.4-2~bpo11+1' for 'bind9' was not found```

**Comment by pmbrull** [comment]
hi @kshinde2512 can you give some info about your setup. This should be working fine AFAIK as tests are running properly from `main`

**Comment by chirag-madlani** [comment]
fixed with #8619
